srcpkgs/font-office-code-pro/template

```diff
@@ -0,0 +1,196 @@
+short_desc="Monospaced sans-serif font for text editors and coding environments"
+maintainer="Jony <maybe-one-day-ubermensch@protonmail.com>"
+license="OFL-1.1"
+homepage="https://github.com/nathco/Office-Code-Pro"
+distfiles="https://github.com/nathco/Office-Code-Pro/archive/${version}.tar.gz"
+checksum="9bca923d17f6c47a586d8e4567d46ccfa58fb8b8e2247b5ee2a19da1597c58f6"
+tags="monospaced sans-serif "
+
+__sz_eot_path="/usr/share/fonts/office-code-pro/slashed-zero/EOT"
+__sz_otf_path="/usr/share/fonts/office-code-pro/slashed-zero/OTF"
+__sz_ttf_path="/usr/share/fonts/office-code-pro/slashed-zero/TTF"
+__sz_woff_path="/usr/share/fonts/office-code-pro/slashed-zero/WOFF"
+__sz_woff2_path="/usr/share/fonts/office-code-pro/slashed-zero/WOFF2"
+
+__dz_eot_path="/usr/share/fonts/office-code-pro/dotted-zero/EOT"
+__dz_otf_path="/usr/share/fonts/office-code-pro/dotted-zero/OTF"
+__dz_ttf_path="/usr/share/fonts/office-code-pro/dotted-zero/TTF"
+__dz_woff_path="/usr/share/fonts/office-code-pro/dotted-zero/WOFF"
+__dz_woff2_path="/usr/share/fonts/office-code-pro/dotted-zero/WOFF2"
+
+# font specific variables
+archs=noarch
+depends="font-util"
+font_dirs="$__sz_eot_path $__sz_otf_path $__sz_ttf_path $__sz_woff_path $__sz_woff2_path "
+font_dirs+="$__dz_eot_path $__dz_otf_path $__dz_ttf_path $__dz_woff_path $__dz_woff2_path"
+
+do_install() {
+	tags+="slashed-zero dotted-zero"
+
+	# slashed zero fonts
+	vmkdir ${__sz_eot_path:1}
+	install -Dm644 Fonts/'Office Code Pro'/EOT/*.eot ${DESTDIR}/${__sz_eot_path:1}
+
+	vmkdir ${__sz_otf_path:1}
+	install -Dm644 Fonts/'Office Code Pro'/OTF/*.otf ${DESTDIR}/${__sz_otf_path:1}
+
+	vmkdir ${__sz_ttf_path:1}
+	install -Dm644 Fonts/'Office Code Pro'/TTF/*.ttf ${DESTDIR}/${__sz_ttf_path:1}
+
+	vmkdir ${__sz_woff_path:1}
+	install -Dm644 Fonts/'Office Code Pro'/WOFF/*.woff ${DESTDIR}/${__sz_woff_path:1}
+
+	vmkdir ${__sz_woff2_path:1}
+	install -Dm644 Fonts/'Office Code Pro'/WOFF2/*.woff2 ${DESTDIR}/${__sz_woff2_path:1}
+
+	# dotted zero fonts
+	vmkdir ${__dz_eot_path:1}
+	install -Dm644 Fonts/'Office Code Pro D'/EOT/*.eot ${DESTDIR}/${__dz_eot_path:1}
+
+	vmkdir ${__dz_otf_path:1}
+	install -Dm644 Fonts/'Office Code Pro D'/OTF/*.otf ${DESTDIR}/${__dz_otf_path:1}
+
+	vmkdir ${__dz_ttf_path:1}
+	install -Dm644 Fonts/'Office Code Pro D'/TTF/*.ttf ${DESTDIR}/${__dz_ttf_path:1}
+
+	vmkdir ${__dz_woff_path:1}
+	install -Dm644 Fonts/'Office Code Pro D'/WOFF/*.woff ${DESTDIR}/${__dz_woff_path:1}
+
+	vmkdir ${__dz_woff2_path:1}
+	install -Dm644 Fonts/'Office Code Pro D'/WOFF2/*.woff2 ${DESTDIR}/${__dz_woff2_path:1}
+}
+
+post_install() {
+	vlicense LICENSE.txt
+}
+
+do_clean() {
+	unset __sz_{eot,otf,ttf,woff,woff2}_path
+	unset __dz_{eot,otf,ttf,woff,woff2}_path
+}
+
+font-office-code-pro-sz-eot_package() {
+	depends="font-util"
+	font_dirs=${__sz_eot_path}
+
+	tags+="slashed-zero"
+
+	pkg_install() {
+		vmkdir ${__sz_eot_path:1}
+		cp -a ${DESTDIR}/${__sz_eot_path:1} ${PKGDESTDIR}/${__sz_eot_path:1}
```

Comment:
Use `vmove`

Reply:
Whenever someone wants the entire font family they can install font-office-code-pro. I copy over the relevant files to the subpackage if someone were to only want one specific part of the font family which was my reasoning for wanting to use cp or vmove

Reply:
Then you'll have file conflicts. You should use `vmove` and the source package depend on the subpackages.

Reply:
I understand now, thanks! I'll refactor.